README.md

```diff
@@ -12,6 +12,10 @@ You don't have to wait for that, you can already use these as custom rules right
 The jpinpoint rules can be run from the command-line using the PMD tool, from your favorite development
 environment with a PMD-plugin, or in SonarQube after packaging them as Sonar plugin.
 
+Documentation of the rules are here:
+* [JavaCodePerformance](docs/JavaCodePerformance.md)
+* [JavaDataAccessPerformance](docs/JavaDataAccessPerformance.md)
+
```

Comment:
Please add the third doc page, as shown under Wiki

Reply:
added